tests/testdata/qgis_server/wms_getfeatureinfo-html-only-with-maptip-raster.txt

```diff
@@ -0,0 +1,4 @@
+*****
+Content-Type: text/html; charset=utf-8
+
+Value Band 5: 90
```

Comment:
Does that qualify as valid "text/html" content type... ? Trying with https://validator.w3.org/#validate_by_input+with_options, I don't think so

Reply:
Right, thanks, addressed per https://github.com/qgis/QGIS/pull/55823/commits/1ca64976314924051efb0a2c3ac44d7b614c7afb

Reply:
hum, that still seems weird to me. IMHO, it shouldn't be up to the user to write the <!DOCTYPE html> stuff in the QGIS <mapTip> element. That should be done by the QGIS WMS code. Well, at least, this is my naive view as a non-export of this part of the code. (and I'm quite surprised that the "Value Band 5: 90" just after <title> is valid, not even encloded in a <body>. But perhaps I'm just too old school with my knowledge of HTML having stopped around year 2000 :-))

Reply:
> IMHO, it shouldn't be up to the user to write the stuff in the QGIS element.

This is possible at least since QGIS 3.4.

https://docs.qgis.org/3.4/de/docs/user_manual/working_with_ogc/server/services.html?highlight=with_maptip

In general it's the responsibility of the user to provide valid content for the maptip -> https://github.com/qgis/QGIS/pull/55751#discussion_r1442087383

> I'm quite surprised that the "Value Band 5: 90" just after <title> is valid

I was also surprised but it's welcome in this case :-)

Reply:
Ahh, just saw your edit - you were referring to the `<!DOCTYPE html>`. I'm going to change it so that's included by the QGIS code.

Reply:
done in https://github.com/qgis/QGIS/pull/55823/commits/d91a0f10887a99b2d0617b7273854baf2f73a4ae